Makefile

```diff
@@ -53,6 +65,42 @@ endif
 SHELL = /usr/bin/env bash -o pipefail
 .SHELLFLAGS = -ec
 
+# Kuadrant component versions
+## kuadrant controller
+#ToDo Pin this version once we have an initial release of the controller
+KUADRANT_CONTROLLER_VERSION ?= latest
+ifeq (latest,$(KUADRANT_CONTROLLER_VERSION))
+KUADRANT_CONTROLLER_GITREF = main
+else
+KUADRANT_CONTROLLER_GITREF = $(KUADRANT_CONTROLLER_VERSION)
+endif
+## authorino
+#ToDo Pin this version once we have an initial release of authorino
+AUTHORINO_OPERATOR_VERSION ?= latest
+ifeq (latest,$(AUTHORINO_OPERATOR_VERSION))
+AUTHORINO_OPERATOR_BUNDLE_VERSION = 0.0.0
+AUTHORINO_OPERATOR_BUNDLE_IMG_TAG = latest
+AUTHORINO_OPERATOR_GITREF = main
+else
+AUTHORINO_OPERATOR_BUNDLE_VERSION = ${AUTHORINO_OPERATOR_VERSION}
+AUTHORINO_OPERATOR_BUNDLE_IMG_TAG = v$(AUTHORINO_OPERATOR_BUNDLE_VERSION)
+AUTHORINO_OPERATOR_GITREF = v$(AUTHORINO_OPERATOR_BUNDLE_VERSION)
+endif
+AUTHORINO_OPERATOR_BUNDLE_IMG ?= quay.io/kuadrant/authorino-operator-bundle:$(AUTHORINO_OPERATOR_BUNDLE_IMG_TAG)
+## limitador
+#ToDo Pin this version once we have an initial release of limitador
+LIMITADOR_OPERATOR_VERSION ?= latest
+ifeq (latest,$(LIMITADOR_OPERATOR_VERSION))
+LIMITADOR_OPERATOR_BUNDLE_VERSION = 0.0.0
+LIMITADOR_OPERATOR_BUNDLE_IMG_TAG = latest
+LIMITADOR_OPERATOR_GITREF = main
+else
+LIMITADOR_OPERATOR_BUNDLE_VERSION = $(LIMITADOR_OPERATOR_VERSION)
+LIMITADOR_OPERATOR_BUNDLE_IMG_TAG = v$(LIMITADOR_OPERATOR_BUNDLE_VERSION)
+LIMITADOR_OPERATOR_GITREF = v$(LIMITADOR_OPERATOR_BUNDLE_VERSION)
+endif
+LIMITADOR_OPERATOR_BUNDLE_IMG ?= quay.io/kuadrant/limitador-operator-bundle:$(LIMITADOR_OPERATOR_BUNDLE_IMG_TAG)
```

Comment:
As above, these should be pinned to a version once there are appropriate versions we can pin here. Installing the "latest" version of a sub component (authorino,limitador, controller) would still be possible by overriding the appropriate env var (AUTHORINO_OPERATOR_VERSION, LIMITADOR_OPERATOR_VERSION,KUADRANT_CONTROLLER_VERSION).